O3-1.tex

```diff
@@ -22,37 +22,40 @@
 
 \title{The LSST Data Management System}
 \author{
-Mario~Juric,$^1$
-Jeffrey~Kantor,$^2$
```

Comment:
Well, the separate commit means I can't just hit the merge button and have to do a manual cherry pick :smile: